### DISCUSSION TITLE
Not able to view Project Settings page

### DISCUSSION THREAD

**Discussion by codingconquerors** [vercel/community#5875] [Help] [answered]
### Question

I want to change Nodejs version for my project but I cannot see anything on the Project Settings page. It is empty. Can somebody guide me?

Thanks,

**Comment by swarnava** [comment]
Hi,

Can you share us a screenshot?

**Reply by ioayoub** [reply]
Hi,  
i have the same issue, here is a screenshot : 
![Screenshot 2024-02-21 at 14 23 37](https://github.com/vercel/community/assets/66207978/1a40e6b5-2bdf-4e4c-8e98-459acbc86db4)

The guilty : 
![Screenshot 2024-02-21 at 14 29 33](https://github.com/vercel/community/assets/66207978/89dc0158-ed78-4ac3-bf73-04dc74986fbe)

**Reply by swarnava** [reply]
Can you share the project link and browser details where you seeing this behavior?

**Reply by ioayoub** [reply]
the account is "ioayoub", same issue with all the projects. Here is an [example](https://vercel.com/ioayoub/next-dashboard/settings)

**Reply by swarnava** [reply, marked as answer]
Thanks! unfortunately, I cannot reproduce this issue. Which web browser are you using? Have you tried on Incognito without extensions enabled?

**Reply by ioayoub** [reply]
Found it ! 
![Screenshot 2024-02-21 at 14 48 24](https://github.com/vercel/community/assets/66207978/dc92f02e-409d-41cb-a7a8-164fc63473b5)

Something happening with this chrome extension 😅

**Reply by ioayoub** [reply]
Thank you for you time :)
